src/infra/sslCmdRepo.go

```diff
@@ -0,0 +1,142 @@
+package infra
+
+import (
+	"errors"
+	"os"
+	"regexp"
+	"strconv"
+
+	"github.com/speedianet/sam/src/domain/dto"
+	"github.com/speedianet/sam/src/domain/valueObject"
+	infraHelper "github.com/speedianet/sam/src/infra/helper"
+)
+
+type SslCmdRepo struct{}
+
+func (repo SslCmdRepo) Add(addSsl dto.AddSsl) error {
+	sslQueryRepo := NewSslQueryRepo()
+
+	httpdVhostsConfig, err := sslQueryRepo.GetHttpdVhostsConfig()
+	if err != nil {
+		matchErr, _ := regexp.MatchString("^(HttpdVhostsConfigEmpty|VhostConfigEmpty)$", err.Error())
+		if !matchErr {
+			return err
+		}
+
+		return errors.New("HttpdVhostsConfigEmpty")
+	}
```

Comment:
Porque nos ajustes do SAM eu vou implementar o _middleware_ de validação de serviço. A ausência do `httpd.conf` e do `vhconf` é causado pela ausência do OLS ou do NGINX instalado. Como eu já iria fazer o _middleware_ na próxima tarefa, então não vi necessidade de tratar de outra forma.

_"The future-proofing of your code should rely on sound design principles rather than superfluous features that are not in use"_

O `httpd.conf` e o `vhconf` poderiam estar vazios por qualquer outro motivo? Sim, poderiam, mas isso não seria um erro 500 que é o padrão que está sendo retornado para o _controller_ e sim um 400 já que esse arquivo de configuração é manipulado de acordo com o usuário. Se ele está mal formatado, significa que foi feito algo de errado pelo próprio usuário. Mas tratar o erro de retorno do _use case_ significa tratar o erro de retorno da infra, então não é a forma correta de se fazer.